test/components/index.test.js

```diff
@@ -1,27 +1,22 @@
 import { describe, it } from 'node:test';
 import assert from 'node:assert';
 import { readFile } from 'node:fs/promises';
-import { basename, dirname } from 'node:path';
-import glob from 'fast-glob';
-
-/**
- * Converts a string to camel case.
- *
- * @param {string} str
- *
- * @return {string}
- */
-const camelCase = (str) => str.replace(/-([a-z])/g, (_match, character) => character.toUpperCase());
 
 describe('components', () => {
   it('re-exports all uswds components', async () => {
     const localComponentsFileContent = await readFile('./build/esm/components/index.js', 'utf-8');
     const localComponentKeys = Array.from(localComponentsFileContent.matchAll(/import ([a-z]+)/gi))
       .map((match) => match[1])
       .sort();
-    const uswdsComponentKeys = glob
-      .sync('./node_modules/@uswds/uswds/packages/*/src/index.js')
-      .map((path) => camelCase(basename(dirname(dirname(path))).replace(/^_?usa-/, '')))
+    const uswdsComponentsIndex = await readFile(
+      'node_modules/@uswds/uswds/packages/uswds-core/src/js/index.js',
+      'utf-8',
+    );
+    const [, exports] = /** @type {RegExpMatchArray} */ (
+      uswdsComponentsIndex.match(/module\.exports = \{([\s\S]+)\};/)
+    );
+    const uswdsComponentKeys = Array.from(exports.matchAll(/(?:\s+([a-zA-Z]+)),/g))
+      .map((match) => match[1])
```

Comment:
can we destructure the regex match?
```suggestion
      .map(([, capturedKey]) => capturedKey)
```

Reply:
We can!